packages/app-mobile/utils/shim-init-react/shimInitShared.ts

```diff
@@ -116,8 +116,7 @@ const shimInitShared = () => {
 		const resourceId = defaultProps.id ? defaultProps.id : uuid.create();
 
 		const ext = fileExtension(filePath);
-		let mimeType = mimeUtils.fromFileExtension(ext);
-		if (!mimeType) mimeType = 'image/jpeg';
+		const mimeType = defaultProps.mime ?? mimeUtils.fromFileExtension(ext) ?? 'image/jpeg';
```

Comment:
For the document scanner, the `mime` `defaultProp` is specified here:
https://github.com/laurent22/joplin/blob/912bf7463facc833de771a97d0f93dac84d09836/packages/app-mobile/components/screens/DocumentScanner/DocumentScanner.tsx#L86

Other [uses of `shim.createResourceFromPath`](https://github.com/search?q=repo%3Alaurent22%2Fjoplin%20createResourceFromPath&type=code) don't seem to provide a `mime` prop.